content/en/docs/Updating/_index.md

```diff
@@ -6,26 +6,49 @@ description: >
  Keeping the theme up to date.
 ---
 
-We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the theme submodule yourself as follows:
+We hope to continue to make improvements to the theme [along with the Docsy community](/docs/contribution-guidelines/). 
+If you have cloned the example site (or are otherwise using the theme as a submodule), you can update the Docsy theme
+yourself. 
 
-1. In your local copy of your project, run:
+Updating Docsy means that your site will build using the latest version of Docsy at `HEAD` and include 
+all the new commits or changes that have been merged since the time when you first added the Docsy submodule, 
+or last updated. If you have any modified content like a Docsy [customiztion](/docs/adding-content/lookandfeel/), those 
+changes will remain in your project. For details about what has changed, see the list of 
+[Docsy commits](https://github.com/google/docsy/commits/master).
+
+## Update a Docsy clone
+
+If you have a local clone of the Docsy theme, you `git pull` all the new changes. For any modifications that you
+made to the theme, you must manually resolve any merge conflicts. 
+
+For example, if `upstream`
```

Comment:
Phrasing is a little awkward. Consider:

For example, here's how you do this if you've set `https://github.com/google/docsy.git` as your `upstream` remote. From the root directory where you cloned the Docsy theme, run:

Reply:
That cleaner, thanks. I guess i was over complicating it by indirectly suggesting that "upstream" could be named anything too (ie `docsy` could be your remote's name - `git pull docsy master`)